mathics/builtin/atomic/symbols.py

```diff
@@ -52,16 +52,19 @@
 from mathics.doc.online import online_doc_string
 
 
-def show_definitions(symbol: Symbol, evaluation: Evaluation) -> list:
+def show_definitions(symbol: Symbol, evaluation: Evaluation) -> List[Expression]:
```

Comment:
Sigh. We are not showing anything we are "gathering", "getting" or "accumulating" things. 

The "show" part is happens only in a REPL.

Reply:
collect?

Reply:
Collect is fine.

Reply:
Now this is called "gather_and_format_definition_rules".